### PR TITLE
change rest serialization examples back to prior state

### PR DESCRIPTION
**Pull request by johnsmartco**

### CODE REVIEW

File: docs-src/stargate-develop/modules/develop/examples/result/rest_curl_get_all.result [hunk 1]
```diff
@@ -13,15 +13,15 @@
...
       "firstname": "Janesha",
-      "evaluations": 
+      "evaluations": [
         {
-          "2019" : "okay",
-          "2020" : "good"
+          "key": 2019,
+          "value": "okay"
         },
-      "top_three_tv_shows": [
```
Comment (tatu-at-datastax) [inline docs-src/stargate-develop/modules/develop/examples/result/rest_curl_get_all.result:21]: Whops. Need to keep this, was accidentally deleted.

Comment (johnsmartco) [reply docs-src/stargate-develop/modules/develop/examples/result/rest_curl_get_all.result:21]: Thanks - fixing..

**Review by tatu-at-datastax — CHANGES_REQUESTED** [review]
Looks good, except for one accidental deletion (I think we caught one during edits but missed one).

File: docs-src/stargate-develop/modules/develop/examples/result/rest_curl_get_all.result (hunk 1 above)
Comment (johnsmartco) [reply docs-src/stargate-develop/modules/develop/examples/result/rest_curl_get_all.result:21]: See most recent commit - thanks.

**Review by tatu-at-datastax — APPROVED** [review]
LGTM

**Comment by johnsmartco** [comment]
@polandll , hi. I'll go ahead and merge.  Tatu and I looked over the changes.  I know you're busy working on many other projects, including the Antora Collector Extension and lots more.  If you spot something in this PR later, I'll create a new one.  Thanks.

**Review by polandll — COMMENTED** [review]
As long as all those brackets are correct, seems like you rolled back correctly.

**Pull request merged**